### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by flywithgeodata**

### CODE REVIEW

File: bat_vis1.Rmd [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: "Bat_vis1"
+author: "Heather S. Davies"
+date: "April 11, 2018"
+output: html_document
+---
+
```
Comment (AJRobillard) [inline bat_vis1.Rmd]: Here is a comment which literally has no substance, but should still demonstrate the comment feature...

File: bat_vis1.Rmd [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+View(batdata)
+
+```
+
+ 
+ 
+ ```{r summary-viz}
+batdata %>% 
+ 
+  group_by(taxonid) %>%
```
Comment (AJRobillard) [inline bat_vis1.Rmd]: This looks like a great plot (I don't really know until I run it though)

File: bat_vis1.Rmd [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+ ```{r summary-viz}
+batdata %>% 
+ 
+  group_by(taxonid) %>% 
+  summarise(l = mean(livingcanopy)) %>% 
+  ggplot(aes(x = taxonid, y = l)) +
+    geom_col() + # a column plot for each entry
+    ggtitle("SJER Distribution of Mean Living Canopy by Taxon Id") +
+    xlab("Taxon ID") + 
+    ylab("Living Canopy (%)") +
```
Comment (AJRobillard) [inline bat_vis1.Rmd]: Should change to "# of Stinging Bees per inch of body"

File: bat_vis1.Rmd (hunk 2 above)
Comment (flywithgeodata) [reply bat_vis1.Rmd]: Thanks - it is a dummy file

File: bat_vis1.Rmd [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+ 
+ ```{r summary-viz}
+batdata %>% 
+ 
+  group_by(taxonid) %>% 
+  summarise(l = mean(livingcanopy)) %>% 
+  ggplot(aes(x = taxonid, y = l)) +
+    geom_col() + # a column plot for each entry
+    ggtitle("SJER Distribution of Mean Living Canopy by Taxon Id") +
+    xlab("Taxon ID") +
```
Comment (AJRobillard) [inline bat_vis1.Rmd]: Change to "Class Member Stung"

File: bat_vis1.Rmd [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+ 
+ 
+ ```{r summary-viz}
+batdata %>% 
+ 
+  group_by(taxonid) %>% 
+  summarise(l = mean(livingcanopy)) %>% 
+  ggplot(aes(x = taxonid, y = l)) +
+    geom_col() + # a column plot for each entry
+    ggtitle("SJER Distribution of Mean Living Canopy by Taxon Id") +
```
Comment (AJRobillard) [inline bat_vis1.Rmd]: Change to "Bee Stings of Classmates by %"

File: bat_vis1.Rmd [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    xlab("Taxon ID") + 
+    ylab("Living Canopy (%)") +
+    theme_minimal() + #change the theme
+    theme(plot.margin=unit(c(.5,1,.5,1), "cm"), #adjust the margin (top, right, bottom, left)
+          axis.text.x = element_text(angle = -60, hjust = -0.1)) # rotate the axis labels
+
```
Comment (AJRobillard) [inline bat_vis1.Rmd]: Add line that changes Taxon labels to student names

File: bat_vis1.Rmd [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+```{r load_data}
+
+batdata <- read.csv("Bat_cty_vir.csv")
+
+View(batdata)
+
+```
+
+
```
Comment (AJRobillard) [inline bat_vis1.Rmd]: Soup Soup Soup

Comment (AJRobillard) [reply bat_vis1.Rmd]:
```{r more bees needed}
batdata
```

**Pull request closed without merging**